src/cdk/a11y/focus-monitor/focus-monitor.ts

```diff
@@ -241,21 +241,22 @@ export class FocusMonitor implements OnDestroy {
       this._windowFocusTimeoutId = setTimeout(() => this._windowFocused = false);
     };
 
+    const eventListenerOptions = supportsPassiveEventListeners() ?
+      {passive: true, capture: true} : true;
+
     // Note: we listen to events in the capture phase so we can detect them even if the user stops
     // propagation.
     this._ngZone.runOutsideAngular(() => {
-      document.addEventListener('keydown', documentKeydownListener, true);
-      document.addEventListener('mousedown', documentMousedownListener, true);
-      document.addEventListener('touchstart', documentTouchstartListener,
-          supportsPassiveEventListeners() ? ({passive: true, capture: true} as any) : true);
+      document.addEventListener('keydown', documentKeydownListener, eventListenerOptions);
+      document.addEventListener('mousedown', documentMousedownListener, eventListenerOptions);
```

Comment:
I'm not sure whether this makes much of a difference since the dev tools don't warn about it, but it wouldn't hurt either.